sdk/communication/communication-job-router/README.md

```diff
@@ -194,11 +213,16 @@ await routerClient.createJob("fdw2rcfb-476e-42f9-a202-b211b41a4ea4", {
 });
 ```
 
-### Receive events for jobs in queue
+## Events
+
+Job Router events are delivered via Azure Event Grid. See our ["subscribe to events" documentation](https://learn.microsoft.com/azure/communication-services/how-tos/router-sdk/subscribe-events) for better understanding of handling events.
+
+In the previous example:
+
+- The job gets enqueued to the “Sales" queue.
+- A worker is selected to handle the job, a job offer is issued to that worker, and a `RouterWorkerOfferIssued` event is sent via Azure Event Grid.
 
-Communication Services Router events are delivered to customers via Azure Event Grid. See [subscribe events](https://learn.microsoft.com/azure/communication-services/how-tos/router-sdk/subscribe-events) on how to setup and handle Router events.
-In above example, the incoming call gets routed to “Sales Queue” queue and a worker is matched to handle the job and an OfferIssued event will arrive at event grid with matched worker information.
-Once you setup to handle events by following steps in above link. Job events will be sent to your endpoint. Json payload of a sample event looks like this:
+Example `RouterWorkerOfferIssued` JSON shape:
 
 ```json
```

Comment:
Can someone confirm this JSON shape is still accurate for a RouterWorkerOfferIssued event please :)